src/main/scala/process/pure/RunNuCommand.scala

```diff
@@ -188,7 +189,7 @@ object RunNuCommand {
        * In the meantime, we silent the logger, and we check for return code of min value with
        * commons use cases:
        */
-      process        <- effect(processBuilder.start())
+      process        <- effectNonBlocking(processBuilder.start())
```

Comment:
That one seems blocking: `processBuilder.start()` calls `factory.createProcess(command, env, processListener, cwd)` which is the platform dependant implementation to create a new process. In that code chain, there is native call, thread, fork, and a lot of other stuff that seems very blocking. How can you be sure that it's non blocking? For example, `Java_java_lang_UNIXProcess_forkAndExec` can't be easely inspected but given the name, I suspect it will create a thread, fork, wait for the fork to happen and return at least the pid or some handler on it.

Reply:
For the record, after inspection, it seems that the call is actually non blocking because what could be blocking seems to be done in an other thread.